jiant/tasks/retrieval.py

```diff
@@ -58,6 +59,7 @@
     "scitail": SciTailTask,
     "snli": SnliTask,
     "socialiqa": SocialIQATask,
+    "spr1": Spr1Task,
```

Comment:
One thing to consider (and I don't have a settled stance on this) is whether each task should be its own implementation. E.g. Both SQuAD v1 and v2 have a single implementation, for cross-lingual transfer tasks (not checked in yet) all language-versions of each task has a single implementation, etc. Can discuss.